docs/DeveloperGuide.adoc

```diff
@@ -295,35 +296,78 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |user |see an overview of all my spending from month to month |better judge my expense/income ratio
 
-|`* * *` |user |add a new person |
+|`* * *` |spendthrift |set a budget for my monthly spending |avoid breaking the bank again
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |lazy user |key in recurring expenses just once |do not have to enter them over and over again
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |careful spender |make month-to-month comparisons of spending in a particular category |see where additional expenses for certain months come from
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |cautious spender |carry over any budget deficits incurred each month to the following month |remember to repay it
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |user who borrows and loans money frequently |track who owes me money/who I owe money to |settle my debts
+
+|`* * *` |SOC student who has to pay school fees regularly |track how much I’ll have to pay and when I have to pay it by |avoid making late payments
+
+|`* * *` |SOC student experienced in writing programs |write my own scripts to manipulate entries in the app |automate and customize the app to my liking
+
+|`* * *` |ambitious user |set goals for my cumulative income |work towards my dream home/car/goal
+
+|`* *` |careless user |undo a mistake |not worry even if I accidentally make an error
+
+|`* *` |careful spender |dedicate different accounts to different holidays |control how much I spend while on the holidays
```

Comment:
For the benefit, will it be better to specify that you are spending on different holidays?